MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -3100,31 +3101,11 @@ private void processNewDayATBScenarios() {
 
     private void processNewDayATBFatigue() {
         boolean inContract = false;
-        for (AtBContract contract : getActiveAtBContracts()) {
-            switch (contract.getMissionType()) {
-                case AtBContract.MT_GARRISONDUTY:
-                case AtBContract.MT_SECURITYDUTY:
-                case AtBContract.MT_CADREDUTY:
-                    fatigueLevel -= 1;
-                    break;
-                case AtBContract.MT_RIOTDUTY:
-                case AtBContract.MT_GUERRILLAWARFARE:
-                case AtBContract.MT_PIRATEHUNTING:
-                    fatigueLevel += 1;
-                    break;
-                case AtBContract.MT_RELIEFDUTY:
-                case AtBContract.MT_PLANETARYASSAULT:
-                    fatigueLevel += 2;
-                    break;
-                case AtBContract.MT_DIVERSIONARYRAID:
-                case AtBContract.MT_EXTRACTIONRAID:
-                case AtBContract.MT_RECONRAID:
-                case AtBContract.MT_OBJECTIVERAID:
-                    fatigueLevel += 3;
-                    break;
-            }
+        for (final AtBContract contract : getActiveAtBContracts()) {
+            fatigueLevel += contract.getContractType().getFatigue();
```

Comment:
If you're planning to move the fatigue level out into the definition files, that's great. Otherwise, not so great. I suppose it's ok if we're intending to leave legacy AtB around.

Reply:
Actually handling fatigue will be part of my retirement, morale, and loyalty project. 

For now it's nice to just isolate it into a single location while reducing the size of Campaign.